common/build.gradle

```diff
@@ -1,12 +1,27 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.google.gradle:osdetector-gradle-plugin:1.6.0'
+    }
+}
+
 plugins {
     id 'java'
     id 'maven'
     id 'com.google.protobuf' version '0.8.5'
 }
 
+apply plugin: 'application'
+apply plugin: 'com.google.osdetector'
+ext.platform = osdetector.os == 'osx' ? 'mac' : osdetector.os == 'windows' ? 'win' : osdetector.os
+
 group = 'network.bisq'
 version = '-SNAPSHOT'
 
+mainClassName = 'bisq.common'
```

Comment:
This isn't the name of a class. Assuming you remove the `application` plugin as suggested above, this should be removed too.